src/net.cpp

```diff
@@ -1544,6 +1544,25 @@ void CConnman::ThreadAddressSeed()
             seed_addrs.erase(std::remove_if(seed_addrs.begin(), seed_addrs.end(),
                                            [](const CAddress& addr) { return !IsReachable(addr); }),
                             seed_addrs.end());
+            Shuffle(seed_addrs.begin(), seed_addrs.end(), FastRandomContext());
+
+            // Make AddrFetch connections to fixed seeds first. This reduces the
+            // load on the fixed seeds that would otherwise be serving blocks for
+            // many new peers requiring IBD.
+            LogPrintf("Initiating AddrFetch connections to fixed seeds.\n");
+            for(size_t addr_pos = 0; addr_pos < 10; addr_pos++) {
+                if(addr_pos >= seed_addrs.size()) {
+                    break;
+                }
+                AddAddrFetch(seed_addrs.at(addr_pos).ToString());
+            }
+            // Give AddrFetch peers some time to provide us with addresses
+            // before adding the fixed seeds to AddrMan
+            if (!interruptNet.sleep_for(std::chrono::minutes(2))) {
+                return;
+            }
+            // As an additional precaution against malicious fixed seeds, add all
```

Comment:
Let's specify what exactly it protects us from?
1. Them not giving us anything
2. Them giving us a lot of fake addrs (I assume our addrman will bucket them together and we won't have to go through every single of them, according to `GetNewBucket`)